CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ Please follow [the Keep a Changelog standard](https://keepachangelog.com/en/1.0.
 
 ## [Unreleased]
 
+## [3.1.0]
+
+### Changed
+
+* remove black from poetry.lock, pyproject.toml, pre-commit, makefile
+* add ruff in upper files
```

Comment:
Whenever you write a changelog, you should only specify changes in public interface. If there are none, then you can be extremely brief:

```suggestion
* Migrate from black to ruff-format
```